index.js

```diff
@@ -54,8 +69,18 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
-
+function getAverageIMDBRating(movies) {
+  allAverages = 0
```

Comment:
Since this variable will represent the total of all IMDb Ratings, I would call this `imdbRatingTotal`.